lib/hounds.js

```diff
@@ -4,6 +4,12 @@ const urlHelper = require('url')
 const Readable = require('stream').Readable
 const Nightmare = require('nightmare')
 
+const consoleLevels = new Map([
```

Comment:
why not just use a regular object?

Reply:
Just a personal preference but basically for the reasons described [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map) (see section "Objects and maps compared").

Reply:
It just limited the node version to 6+ (but according to this it's been there since 4.x http://node.green/).

Reply:
Basically use maps when the values are all the same type or objects that have the same properties.

Reply:
OK - I think given where node is at now and ES6 features we should just go ahead. (I understand the general motivation - I was just being cautious about using new features without much gain)

Reply:
Yeah it's ES6 which is like 2015 :)